kuma/wiki/views/document.py

```diff
@@ -1107,3 +1107,43 @@ def _document_api_PUT(request, document_slug, document_locale):
         resp.status_code = 205
 
     return resp
+
+
+@never_cache
+@require_GET
+def doc_api(request, locale, path):
```

Comment:
I'm wondering if we should move this into a new file `kuma/wiki/api/v1/view.py`? Besides the fact that this function has a name similar enough to `document_api` to perhaps be a little confusing, I'm thinking that it would be nice to separate this into its own file so that we could more easily move it later (in another PR). For example, if we decide, as @jwhitlock suggested [here](https://github.com/mozilla/kuma/pull/5285#pullrequestreview-211928950), to gather all of the `/api/v1` endpoints within their own folder like `/kuma/api/v1`.

Reply:
I was thinking also that perhaps `slug` would be a better name than `path`, only because it matches our current usage?

Reply:
I'll do both of those things. I started with a views.py file, then thought I was going to end up using utility functions defined in document.py, so moved it there. But the implementation turned out to be simpler, so now there is no reason not to have it in its own file. Thanks for pointing that out.